### ISSUE TITLE
Plots not generating using Windows 10 Home

### ISSUE THREAD

**Issue by jennie0530** [jasp-stats/jasp-issues#508]
* JASP version: 0.10.2
* OS name and version: Windows 10 Home
* Analysis: ANOVA, Descriptive Statistics, pretty much any of the plots
* Bug description: None of the plots are not generated. It used to work the first few times and after some time, the plots stopped working..
* Add JASP files as a zip:
[HSB JASP BUG REPORT.zip](https://github.com/jasp-stats/jasp-issues/files/3672865/HSB.JASP.BUG.REPORT.zip)
* Screenshot: 
![plot bug screenshot 2](https://user-images.githubusercontent.com/55999416/65908355-e72dab80-e38b-11e9-9c77-f7cb5438c708.png)
![plot bug screenshot](https://user-images.githubusercontent.com/55999416/65908356-e72dab80-e38b-11e9-9d7b-e0c2b3889573.png)

* Expected behaviour: Plot generation
* Steps to reproduce: 
1. Go to 'Descriptive Statistics'
2. Click on 'Plots - Distribution plots, Boxplots, Q-Q plots'
3. See error

**Comment by boutinb** [comment]
Hi @jennie0530 We have used your JASP file, and cannot reproduce your problem. What do you mean exactly with: it works a few time, and then not anymore, what did you do exactly? Did you save the JASP file, and reopen it afterwards and then you did not see the plots anymore?
Could you also send us the log files: Go to the File menu, Preferences, Advances, Logging options, check the 'Log to file' checkox, and redo your reproduction path.
Kind regards,
Bruno

**Comment by jennie0530** [comment]
Hi,

 

Here are the log files as well as the data and jasp files. What I meant was, when I ran JASP for the first few times the plots were generated with no problem regardless of what data I was using.

Now, no matter which data set I run JASP with, none of the plots are generated. It just shows a blank. What I did was, I opened a data set and ran an ANOVA with Q-Q plot of residuals for normality check. Also, I ran descriptive statistics with Distribution plots, Boxplots, and Q-Q plots. But none of the plots are generated in the output.

 

Please help.

 

Best regards,

Jennie

**Comment by jennie0530** [comment]
> Hi, Here are the log files as well as the data and jasp files. What I meant was, when I ran JASP for the first few times the plots were generated with no problem regardless of what data I was using. Now, no matter which data set I run JASP with, none of the plots are generated. It just shows a blank. What I did was, I opened a data set and ran an ANOVA with Q-Q plot of residuals for normality check. Also, I ran descriptive statistics with Distribution plots, Boxplots, and Q-Q plots. But none of the plots are generated in the output. Thanks. [JASP Bug.zip](https://github.com/jasp-stats/jasp-issues/files/3688727/JASP.Bug.zip) [Logs.zip](https://github.com/jasp-stats/jasp-issues/files/3688728/Logs.zip)

**Comment by jennie0530** [comment]
I already tried reinstalling JASP as well but it's still not working :(

**Comment by boutinb** [comment]
I see that the JASP file contains the plots. And if I open your JASP file, I can see the plots.
I saw that you use 0.10.2. We have released a new version 0.11. And very soon 0.11.1. Can you try the latest version (http://static.jasp-stats.org/Nightlies/Windows/JASP-nightly-WindowsJaspResultsHotfix-9eace6e10febcc3b6a3d6a61df209168eaad6876.msi) ?

**Comment by jennie0530** [comment]
I tried downloading the file you sent me and the one on the website but the
plots are still not showing (I used jasp file.) Also, now the new version
says it doesn't support sav files anymore. Is this correct?

Thanks for your help.

**Comment by boutinb** [comment]
Hi @jennie0530,
This is really strange...
I want to ask you something that might be a bit technical, but let's try...
After having opened a JASP file that should contain a plot, with your Window Explorer, go to the folder with your user account ('c:\Users\Jennie' if your Windows account jennie is ).
Then click on the View menu of Window Explorer, and check the 'Hidden items' checkbox: you should see then the AppData folder.
In this AppData folder, go to Local\JASP\temp. Here you should see 1 folder as number. Go in, then you should see the folder resources, and the 0 folder.
Can you see in this 0 folder a png file with the plot should see in JASP?

**Comment by jennie0530** [comment]
I followed your steps and I do see a plot in the resources folder. But it’s not showing in the results from JASP itself.

**Comment by boutinb** [comment]
Hi @jennie0530,
OK, the png files are at the right positions. Now I would like to know why JASP cannot read them. For this, I will ask you again to do something quite technical, but I hope I can then get a hint why it does not work on your machine:

- Start a Command prompt on your PC: Click on the Search (magnify glass icon), type `cmd`, and press enter: the Command Prompt should now open.
- In this Command prompt, type: `cd "\Program Files\JASP"` and press enter.
- Then type `.\JASP --remote-debugging-port=1234`: JASP should start.
- Then start Google Chrome, and navigate to `localhost:1234`: you should see 2 'inspectable' pages.
- Click on the first link, titled `JASP`: you should see what we call the Chrome Inspector
- In this Inspector, choose the Console menu at the top menu bar: you should see a big `Hi.`.
- Then, in JASP run your analysis, and make a plot (that does not appear).
- Go back to Google Chrome: do you see a red error message? If yes, could you send it to me? 

Thank you really much!

**Comment by jennie0530** [comment]
Hi,

 

I followed your instructions but nothing showed up on the Chrome Inspector and the plots are still not showing up. The png files are still in the temp folder as before. 

 

Thank you for your help as always. I really hope everything will be fixed soon. 

 

Best regards,

Jennie

**Comment by boutinb** [comment]
Hi @jennie0530 
Could you do the same steps to open the console menu of the Chrome Inspector, but this time, type:
- $(".jasp-image-image")[0].style.backgroundImage
This should give you the URL of the (first) png file, something like that:
"url("file:////Users/brunoboutin/.JASP/temp/92393/resources/0/_0.png?x=0.5612007586922116")"
Then you could in a new tab of Chrome, copy-paste this URL:
file:////Users/brunoboutin/.JASP/temp/92393/resources/0/_0.png?x=0.5612007586922116
Can you see your PNG file?
Best regards,
Bruno

**Comment by jennie0530** [comment]
Hi,

 �

I tried typing $(".jasp-image-image")[0].style.backgroundImage from the console menu of the Chrome Inspector but nothing happens the line just disappears as soon as I press enter. I even tried typing this in cmd as well (just in case? I don’t really know what I’m doing so just tried whatever) but it just says ‘$’ is not recognized as an internal or external command, operable program or batch file.

 �

When I check the file location of the png file it’s still there. I see the plots in png file in the temp folder (_0.png) but still not in the JASP output nor does any URL shows up in the console menu of the Chrome Inspector..

 �

 �

**Comment by jennie0530** [comment]
P.s. This is what it looks like when I type the below and the ".jasp-image-image" is in red. And then when I press enter nothing shows up and the field goes blank.

 �

 �

 �

**Comment by boutinb** [comment]
Hi Jennie,
This is really difficult to check what you do and to understand what happens. Maybe the best way would be to skype. Is it an option for you?

**Comment by moinhotm** [comment]
Hi, I'm going through the same problem. I'm using the latest version of JASP and it's performance is very unstable. Many things, especially images, won't load! I tried looking for the images in the temp folder, but the folders were empty. 

What I noticed that helps is asking for JASP to do other graphs. When I ask for other things, what I had previously selected finally works. For example, I asked for a distribution plot, but it didn't load completely/show the plot. I then asked to also show a boxplot. When I do that, it refreshes and the distribution plot appears, but now the boxplot is just blank. If I ask for something else, then sometimes the boxplot will load too. It's so annoying! :(
